Reduce visibility of Tranquility documentation (#11134)
apache/druid: docs/ingestion/index.md

  ~ under the License.
  -->

All data in Druid is organized into _segments_, which are data files each of which may have up to a few million rows.
Loading data in Druid is called _ingestion_ or _indexing_, and consists of reading data from a source system and creating
segments based on that data.

In most ingestion methods, the Druid [MiddleManager](../design/middlemanager.md) processes

> (or the [Indexer](../design/indexer.md) processes) do the work of loading data. One exception is

Reviewer: ```suggestion
(or the [Indexer](../design/indexer.md) processes) load your source data. One exception is
```
nit
